src/js/shim.js

```diff
@@ -4,17 +4,45 @@
  */
 "use strict";
 
-[(function(exports) {
-
+const exports = {};
 /**
  * we load these lazily.
  */
 
+import {FakeMessages, FakeDisk, Connects} from './fakes.js';
+
+function wrapObject(base) {
```

Comment:
I added features to this thing that I'm barely using. It's handy and I should document it.